spec/factories/gias_schools.rb

```diff
@@ -24,5 +24,11 @@
   factory :gias_school do
     sequence(:urn) { _1 }
     name { "Hogwarts" }
+    ukprn { "fake_uprn" }
```

Comment:
At this point I think we should use Faker or sequence. Otherwise if someone uses `create_list(:gias_school, 5)` we'll end up with 5 GiasSchools with the same attributes.